### PR TITLE
[TASK] Revise Gibuilder/Gifbuilder chapter

### PR DESCRIPTION
**Pull request by brotkrueml**
Releases: main, 12.4, 11.5

### CODE REVIEW

File: Documentation/Gifbuilder/Gifbuilder/Index.rst [hunk 1]
```diff
@@ -1,73 +1,75 @@
...
-.. _gifbuilder:
+..  include:: /Includes.rst.txt
+..  index:: GIFBUILDER
+..  _gifbuilder:
 
 ==========
 GIFBUILDER
 ==========
 
 GIFBUILDER is an object type, which is used in many situations for creating
```
Comment (linawolf) [inline Documentation/Gifbuilder/Gifbuilder/Index.rst:9]:
```suggestion
:typoscript:`GIFBUILDER` is an object type, which is used in many situations for creating
```
Should we mark gifbuilder as TypoScript throughout as that is a word from TypoScript and not from english?

File: Documentation/Gifbuilder/Gifbuilder/Index.rst [hunk 2]
```diff
@@ -1,73 +1,75 @@
...
+
+On using the special function :typoscript:`max()`, the maximum of multiple
+values can be determined. Example:
+
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+
+    XY = [10.w]+[20.w], max([10.h], [20.h])
+
+Here's a full example (taken from "styles.content (default)" of older TYPO3
```
Comment (linawolf) [inline Documentation/Gifbuilder/Gifbuilder/Index.rst:47]: Not sure how helpful this example is?

Comment (brotkrueml) [reply Documentation/Gifbuilder/Gifbuilder/Index.rst:47]: At least it is an example. I never worked with GIFBUILDER in the past, so have not really a clue what is going on here. The whole working from me on GUIFBUILDER now is preparing it for adding a changelog entry for v13 (support of webp).

The example can be tackled in a separate PR.

File: Documentation/Gifbuilder/Gifbuilder/Index.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/Gifbuilder/Gifbuilder/Index.rst:9]: Thought also about it, but there are different mentions of GIFBUILDER, sometimes referencing as "GIFBUILDER object". At first glance, I don't know if it is referring to the TypoScript cObject or something else. So I stpped away from marking all the GIFBUILDER occurrences as TypoScript for now.

File: Documentation/Gifbuilder/Gifbuilder/Index.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/Gifbuilder/Gifbuilder/Index.rst:47]: ack

File: Documentation/Gifbuilder/Gifbuilder/Index.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/Gifbuilder/Gifbuilder/Index.rst:9]: I think it is the cObject, all cObjects are classes under the hood and therefore can also be called object

Comment (brotkrueml) [reply Documentation/Gifbuilder/Gifbuilder/Index.rst:9]: @linawolf Do you know, why sometimes `GIFBUILDER` is used, and once `->GIFBUILDER`? Is it the same? I also saw this with, for example, `->if`  and `if`.

Comment (linawolf) [reply Documentation/Gifbuilder/Gifbuilder/Index.rst]: I believe one referes to the TYPOSCRIPT Object and one to the corresponding function in PHP? But maybe also no one thought about it?

Comment (brotkrueml) [reply Documentation/Gifbuilder/Gifbuilder/Index.rst]: Don't know, leave it as is for now.

Comment (linawolf) [reply Documentation/Gifbuilder/Gifbuilder/Index.rst]: Ack

**Review by linawolf — APPROVED** [review]

**Pull request merged**